src/FormField/Calendar.php

```diff
@@ -55,13 +55,17 @@ public function renderView()
         if ($format = $this->app->ui_persistence->$typeFormat) {
             $formatter = "function(date, settings){
                             if (!date) return;
-                            return atk.phpDate('${format}', date);
+                            return atk.phpDate([format], date);
                         }";
-            $this->options['formatter'][$this->type] = new jsExpression($formatter);
+            $this->options['formatter'][$this->type] = new jsExpression($formatter, ['format' => $format]);
         }
 
         $this->options['type'] = $this->type;
 
+        if ($dayOfWeek = $this->app->ui_persistence->firstDayOfWeek) {
```

Comment:
while 0 (Sunday) is default for JS library then this will work, otherwise you have to validate it with `isset` or `is_int` here.

Reply:
Yes, default is 0 for Calendar options, so will start on Sunday like the actual implementation.